lib/ansible/modules/user.py

```diff
@@ -70,8 +70,8 @@
             - Optionally set the user's shell.
             - On macOS, before Ansible 2.5, the default shell for non-system users was C(/usr/bin/false).
               Since Ansible 2.5, the default shell for non-system users on macOS is C(/bin/bash).
-            - On other operating systems, the default shell is determined by the underlying tool being
-              used. See Notes for details.
+            - On other operating systems, the default shell is determined by the underlying tool
+              (see Notes for details) being used.
```

Comment:
```suggestion
            - See notes for details on how other operating systems determine the default shell by the underlying tool.
```

Reply:
@Akasurde I have done changes as you suggested